src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -36,6 +36,103 @@ public static partial class Kernel32
         /// </summary>
         public const int NMPWAIT_NOWAIT = 0x00000001;
 
+        /// <summary>Hardware-dependent cursor resource.</summary>
+        public static readonly unsafe char* RT_CURSOR = MAKEINTRESOURCE(1);
```

Comment:
Yes, that's a good point. Either way (same partial or separate) is fine. Your call.